saleor/graphql/webhook/tests/mutations/test_delivery_retry.py

```diff
@@ -0,0 +1,102 @@
+from unittest.mock import patch
+
+import graphene
+
+from saleor.graphql.tests.utils import assert_no_permission, get_graphql_content
```

Comment:
Just styling thing, but I see that you use relative import in different files, and here an absolute. Maybe we could align it to one approach?